src/main/java/com/minecolonies/coremod/colony/crafting/CustomRecipe.java

```diff
@@ -142,40 +144,59 @@
      */
     private int maxBldgLevel = 5;
 
-
     /**
      * This class can only be created by the parse static
      */
     private CustomRecipe()
     {
+    }
 
+    private static ItemStack idToItemStack(String itemId)
+    {
+        final int tagIndex = itemId.indexOf("{");
+        final String tag = tagIndex > 0 ? itemId.substring(tagIndex) : null;
+        itemId = tagIndex > 0 ? itemId.substring(0, tagIndex) : itemId;
+        final String[] split = itemId.split(":");
+        final Item item = ForgeRegistries.ITEMS.getValue(new ResourceLocation(split[0], split[1]));
```

Comment:
should we check if it returns an empty air item here?  And log an error when parsing fails, also for the tags maybe.

Reply:
I'll log a warning for air/empty, and errors for parse failures. I'm not going to refer them back to us, since this is likely to be modpack author issues.

Reply:
ye thats fine, just nice to have to the ppl modifying it see if sth went wrong